subsys/net/lib/sockets/getaddrinfo.c

```diff
@@ -226,6 +226,23 @@ int z_impl_z_zsock_getaddrinfo_internal(const char *host, const char *service,
 		st1 = DNS_EAI_SYSTEM;
 	}
 
+	if (family == AF_UNSPEC && IS_ENABLED(CONFIG_NET_IPV6)) {
```

Comment:
Would be nice to get a comment here what exactly happens here (i.e. that if family == AF_UNSPEC, we already have queried IPv4 case in the previous block).